app/Models/Feed.php

```diff
@@ -17,6 +17,11 @@ class FreshRSS_Feed extends Minz_Model {
 	 * @var int
 	 */
 	const KIND_HTML_XPATH = 10;
+	/**
+	 * Normal XML with XPath scraping
+	 * @var int
+	 */
+	const KIND_XML_XPATH = 15;
```

Comment:
I agree when there is a mix of constants with different visibilities, or potentially the first time we introduce a constant in a class. Otherwise it does not matter as they are public by default, just like classes, and I would rather keep the same syntax than the other surrounding constants.